scripts/o3de/o3de/project_properties.py

```diff
@@ -104,24 +109,43 @@ def edit_project_props(proj_path: pathlib.Path = None,
         proj_json['summary'] = new_summary
     if new_icon:
         proj_json['icon_path'] = new_icon
-    if new_tags:
-        tag_list = new_tags.split() if isinstance(new_tags, str) else new_tags
-        proj_json.setdefault('user_tags', []).extend(tag_list)
-    if delete_tags:
-        removal_list = delete_tags.split() if isinstance(delete_tags, str) else delete_tags
-        if 'user_tags' in proj_json:
-            for tag in removal_list:
-                if tag in proj_json['user_tags']:
-                    proj_json['user_tags'].remove(tag)
-                else:
-                    logger.warning(f'{tag} not found in user_tags for removal.')
-        else:
-            logger.warning(f'user_tags property not found for removal of {delete_tags}.')
-    if replace_tags:
-        tag_list = replace_tags.split() if isinstance(replace_tags, str) else replace_tags
-        proj_json['user_tags'] = tag_list
-    # Update the gem_names field in the project.json
-    _edit_gem_names(proj_json, new_gem_names, delete_gem_names, replace_gem_names, is_optional)
+    if new_version:
```

Comment:
Should we update this to use the same `if isinstance(new_version, str)` check as the gem_properties.py?

Reply:
good catch